Formula/global.rb

```diff
@@ -117,6 +124,18 @@ def pyfunc ():
       assert_no_match(/test\.py/, shell_output("#{bin}/global -r py2func # correctly fails"))
       assert_no_match(/test\.py/, shell_output("#{bin}/global -s pyvar   # correctly fails"))
     end
+    if build.with? "universal-ctags"
```

Comment:
Probably not worth adding this; these tests are only for CI and thus will never be run with options so this code will never be run.